icu4c/source/test/fuzzer/fuzzer_driver.cpp

```diff
@@ -10,11 +12,49 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size);
 
 int main(int argc, char* argv[])
 {
-  (void) argc;
-  (void) argv;
-  const char *fuzzer_data = "abc123";
-   
-  LLVMFuzzerTestOneInput((const uint8_t *) fuzzer_data, strlen(fuzzer_data));
+  bool show_warning = true;
+#if UPRV_HAS_FEATURE(address_sanitizer)
+  show_warning = false;
+#endif
+#if UPRV_HAS_FEATURE(memorys_sanitizer)
+  show_warning = false;
+#endif
+  if (argc > 2 && strcmp(argv[2], "-s") == 0) {
+    show_warning = false;
+  }
+  if (show_warning) {
+      std::cout << "WARNING: This binary work only under build configure with" << std::endl
+                << " CFLAGS=\"-fsanitize=$SANITIZE\""
+                << " CXXFLAGS=\"-fsanitize=$SANITIZE\""
+                << " ./runConfigureICU ... " << std::endl
+                << "  where $SANITIZE is address or memory" << std::endl
```

Comment:
I would put quotes around "address" and "memory" to make it clear that you mean exactly those words.